pulseapi/profiles/test_views.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from django.core.urlresolvers import reverse
+
+from pulseapi.tests import PulseMemberTestCase
+
+
+class TestProfileView(PulseMemberTestCase):
+    def test_get_single_profile_data(self):
+        """
+        Check if we can get a single profile by its `id`
+        """
+        id = self.users_with_profiles[0].id
+        response = self.client.get(reverse('profile', kwargs={'pk': id}))
+        self.assertEqual(response.status_code, 200)
+
+    def test_profile_data_serialization(self):
+        """
+        Make sure profiles have "created_entries" array
+        """
+        id = self.users_with_profiles[0].id
+        response = self.client.get(reverse('profile', kwargs={'pk': id}))
+        entriesjson = json.loads(str(response.content, 'utf-8'))
+        self.assertGreater(len(entriesjson['created_entries']), 0)
```

Comment:
this is risky since the len can be an arbitrary value and still pass. We should assert that what is being passed back from the route is what's actually in the db.

Try:
```py
created_entries = []
entry_creators = OrderedCreatorRecord.objects.filter(creator__profile=self.users_with_profiles[0])
for entry_creator in entry_creators:
    created_entries.append(EntrySerializer(entry_creator.entry).data)

self.assertEqual(entriesjson['created_entries'], created_entries)
```